.local/matrix-runner-4.sh

```
@@ -0,0 +1,178 @@
+  date -u +"%Y-%m-%dT%H:%M:%SZ start" | tee -a "$OUT/matrix.log"
+
+  teardown_redis_ns
+  deploy_redis_ns
+
+  local k6pod=""
+  for i in $(seq 1 120); do
+    k6pod=$(kubectl -n redis get pods -l app.kubernetes.io/name=loadgen -o jsonpath='{.items[0].metadata.name}' 2>/dev/null)
+    [ -n "$k6pod" ] && break
+    sleep 1
+  done
+  for i in $(seq 1 180); do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)